### DISCUSSION TITLE
Stability issues with NVIDIA gpu on linux?

### DISCUSSION THREAD

**Discussion by shi-yan** [gpuweb/gpuweb#3545] [Q&A]
I briefly tried chrome's webgpu implementation on linux using NVIDIA 1080. It's not stable compared to the Mac implementation.

I saw weird artifacts and the device can drop off sometimes. 

Do you see the same? has the situation been reported and worked on?

**Comment by Kangz** [comment]
This sounds like a Chromium issues, so ideally you should report it on https://crbug.com/dawn/new with your `about:gpu` information and a repro case. Note however that proper Linux support is not in the targets for the first release of WebGPU in Chromium and would come only a bit later.

**Comment by Frendysemon** [comment]
Yeah NVIDIA on Linux can be a bit hit or miss with WebGPU especially on older cards like the GTX 1080. I would try the latest NVIDIA driver first and see if the issue still happens.
